src/PokeDetail.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+
+class PokeDetail extends Component {
+    state = { data:{} };
+
+    loadPokeData = async () => {
+        const { id } = this.props.match.params;
+        const url = `https://pokedex-alchemy.herokuapp.com/api/pokedex/${id}`;
+        const resp = await fetch(url);
+        const data = await resp.json();
+        this.setState({ data })
+    };
+
+    componentDidMount() {
+        this.loadPokeData();
+    }
+
+    render() { 
+        const { data } = this.state;
+        return ( 
+            <section>
```

Comment:
Just missing your loading spinner here while your data is fetching from the API.